toml.py

```diff
@@ -25,6 +25,10 @@ def utcoffset(self, dt):
     def dst(self, dt):
         return datetime.timedelta(0)
 
+class InlineTableDict(dict):
+    def __init__(self, *args):
+        dict.__init__(self, args)
+
```

Comment:
`_dict` applies to `load` and `loads`
This class applies to `dump` and `dumps`. I don't see this as a big issue.
Feel free to submit a follow-up PR.